package.json

```diff
@@ -16,6 +16,7 @@
     "brfs": "^1.0.0",
     "standard": "^6.0.5",
     "tape": "^4.0.0",
+    "webtorrent-fixtures": "rom1504/webtorrent-fixtures#add_parse_torrent_file_fixtures",
```

Comment:
This should be moved back to a version once the PR in webtorrent-fixtures is merged and released. Just for testing.